public/js/index.js

```diff
@@ -0,0 +1,24 @@
+//Youtube slideshow for index page
+function onGoogleLoad() {
+  gapi.client.setApiKey('AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI');
```

Comment:
Ah that should be okay then although if I remember right you can get playlist videos without an api key